src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/NewLateBinding.vb

```diff
@@ -265,19 +269,22 @@ Namespace Microsoft.VisualBasic.CompilerServices
 
         <RequiresUnreferencedCode(LateBindingTrimMessage)>
         Private Shared Function InternalLateIndexGet(
-                ByVal instance As Object,
```

Comment:
Shouldn't these be lower case?

Reply:
A majority of the VB code is upper case for locals. I was just trying to follow the most recent style. Happy to update them if that is appropriate.

Reply:
Consider reverting the uppercase changes. It looks like a significant fraction of the locals in this file still use lowercase, and it would be good to limit the diffs to just essential changes.